starcli/search.py

```diff
@@ -124,13 +130,9 @@ def search(
     if debug:
         print("DEBUG: search: url:", url)  # print the url when debugging
 
-    try:
-        repositories = requests.get(url).json()  # get the response using the url
-    except requests.exceptions.ConnectionError:
-        secho("Internet connection error...", fg="bright_red")
-        return None
+    request = get_valid_request(url)
```

Comment:
we should `return None` after this line because when "Internet connectivity error..." is printed None is returned therefore in this scenario `return request.json()["items"]` will throw an error
```
return request.json()["items"]
AttributeError: 'NoneType' object has no attribute 'json'
```
same thing for line 150

Reply:
note that the error is shown when I tried this without wifi connection

Reply:
I've modified the methods to return None if request is None. Sorry about the additional commits, my local repo got a little messed up and it took a bit of work to fix it. No worries about the delay. My bad about the delay on my end, I missed the notification from yesterday.